source/vocab/relations.ttl

```diff
@@ -446,7 +447,7 @@ prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubTyp
     rdfs:domain :SeriesMembership ;
     rdfs:subPropertyOf :object ;
     sdo:rangeIncludes :Instance ;
-    rdfs:label "ingår i serie"@sv .
+    rdfs:label "medlemskap i serie"@sv .
```

Comment:
Why change label here? for me it introduces a confusion with seriesMembership label.

Reply:
It came from a note I had (from some chat IIRC) about differentiating `inSeries` and `partOfSeries` through its label. But if we don't need to I'll gladly revert this label, since when _used_ its context (i.e. the type of the thing it's placed on  (`:SeriesMembership`)) should make the simpler label obvious, and when listed the domain/range context is reasonably needed anyway.

Reply:
Sounds good. We should do a follow up on that note on any possible differences, and if so document that in a comment or scopenote at least.